src/Driver.php

```diff
@@ -161,10 +161,26 @@ public function click($selector)
         return $this;
     }
 
-    public function setAcceptLanguage(array $langs)
+    public function clickXpath($selector)
     {
+        $this->script .= "
+var x = require('casper').selectXpath;
```

Comment:
I don't really know how these functions are used, but maybe it could be useful to isolate them with an IIFE:
`(function() { var x = ... }());`